src/templates/Footer/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Button from 'atoms/Button';
+import Icon from 'atoms/Icon';
+import { Layout, Col } from 'atoms/Layout';
+import LinkWrapper from 'atoms/LinkWrapper';
```

Comment:
I was thinking -- does it make sense to leverage Josh's context script for the npm publish to create a master index.js that we can just import these from? Instead of having 4 lines of imports, just have one?